source/hardware/controllers/pioneer_ddj_sb3.rst

```diff
@@ -0,0 +1,381 @@
+2         :hwlabel:`SHIFT` + Pad                                           Stop sample 2
+3         Pad                                                              Play sample 3
+3         :hwlabel:`SHIFT` + Pad                                           Stop sample 3
+4         Pad                                                              Play sample 4
+4         :hwlabel:`SHIFT` + Pad                                           Stop sample 4
+5         Pad                                                              Load sample 1
+5         :hwlabel:`SHIFT` + Pad                                           Unload sample 1
+6         Pad                                                              Load sample 2
+6         :hwlabel:`SHIFT` + Pad                                           Unload sample 2
+7         Pad                                                              Load sample 3
+7         :hwlabel:`SHIFT` + Pad                                           Unload sample 3
+8         Pad                                                              Load sample 4
+8         :hwlabel:`SHIFT` + Pad                                           Unload sample 4
+========  ===============================================================  ==========================================
+
+Trans Mode
+^^^^^^^^^^
+Trans mode is enabled by holding :hwlabel:`SHIFT` down and by pressing the :hwlabel:`SAMPLER` button. The button will
+blink when selected. Press one of the pads to trigger one of the built in cut effects.
+
+.. figure:: ../../_static/controllers/pioneer_ddj_sb3_trans.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-SB3 (performance pads trans mode)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-SB3 (performance pads in trans mode)
+
+   ========  ===============================================================  ==========================================
+   No.       Control                                                          Function
+   ========  ===============================================================  ==========================================
+   1         Pad                                                              1/16 beat cut effect
+   2         Pad                                                              1/8 beat cut effect
+   3         Pad                                                              1/4 beat cut effect
+   4         Pad                                                              1/2 beat cut effect
+   5         Pad                                                              1 beat cut effect
+   6         Pad                                                              2 beat cut effect
+   7         Pad                                                              1/3 beat cut effect
+   8         Pad                                                              Manual cut effect
```

Comment:
What does this mean?

Reply:
When you press and hold down the pad in this mode, the fader for that channel gets set to min and max at that rate. So, every beat it sets the fader from max to min and so on with the beat. It's a controller built-in feature.

Reply:
Here's what it looks like. I've seen it called a cut effect
![cut effect](https://user-images.githubusercontent.com/55333625/147422108-7ecdd2d3-4a62-495f-8fec-f377e5f339ed.gif)
.

Reply:
Please describe that in the manual; that's not obvious or common knowledge.

Reply:
updated the description, lmk if it needs more work